gdk_pixbuf2/README.md

```diff
@@ -4,9 +4,9 @@ Ruby/GdkPixbuf2 is a Ruby binding of GdkPixbuf-2.x.
 
 Requirements
 ------------
-   Ruby: http://www.ruby-lang.org/
-   GTK+: http://www.gtk.org/
-   Ruby/GLib2, Ruby/GTK2: http://ruby-gnome2.sourceforge.net/
+*  Ruby: http://www.ruby-lang.org/
+*  GTK+: http://www.gtk.org/
+*  Ruby/GLib2, Ruby/GTK2: http://ruby-gnome2.sourceforge.net/
```

Comment:
Can you remove a space between `*` and the next character?
We need only one space after `*`.

Reply:
Thank you. I remove spaces.